route/osrm/client.go

```diff
@@ -128,6 +131,11 @@ func (c *client) get(uri string) (data []byte, err error) {
 	var key string
 
 	if c.useCache {
+		// sha1 is used to shorten the key for faster cache lookup than directly using the lenthy uri as key.
+		// The chance of hash colision is extremely low for sha1.
+		// The cache is local to the user, which won't become a secrity threat even when key colides.
```

Comment:
```suggestion
		// The cache is local to the user, which won't become a security threat even when key collides.
```
Just a small typo. :relaxed:

Reply:
Fixed. Good eyes!